tests/lib/cmsis_dsp/filtering/testcase.yaml

```diff
@@ -8,7 +8,13 @@ tests:
     filter: ((CONFIG_CPU_AARCH32_CORTEX_R or CONFIG_CPU_CORTEX_M) and CONFIG_FULL_LIBC_SUPPORTED
       ) or CONFIG_ARCH_POSIX
     tags: cmsis-dsp
-    skip: true
+    platform_allow:
+      - native_sim/native
+      - mps2/an385
+      - mps2/an521/cpu0
+      - qemu_cortex_r5/zynqmp_rpu
```

Comment:
can you explain why platform_allow is used here (and in the following files)? I see that other scenarios from this yaml (and in other yamls modified with this PR) are not using platform allow. Isn't it a case, where some combination of platform_key would prevent over testing in the CI and won't block tests execution on platform not among the allow list (even though executing those might not bring any added value, but some people still would want to execute them on their platforms)?

Reply:
This is already explained in the commit. This is testing a feature, not platforms and this is already good coverage, if you think you have a better way to do that and expand this, you can submit a followup, but right now this is 1000000 better than being completely skipped.